package.json

```diff
@@ -14,9 +14,9 @@
     "node": ">=4"
   },
   "scripts": {
-    "transpile": "node_modules/.bin/babel src -d build",
-    "transpile-watch": "node_modules/.bin/babel src -d build -w",
-    "postinstall": "yarn transpile",
+    "prepublish": "yarn transpile",
+    "transpile": "node_modules/.bin/babel src -d dist",
+    "transpile-watch": "node_modules/.bin/babel src -d dist -w",
```

Comment:
Let's remove babel from the dependencies and move it to the dev dependencies. Also let's define chai as peer dependency.